client/app/visualizations/chart/chart-editor.html

```diff
@@ -263,3 +267,35 @@
       </tbody>
     </table>
   </div>
+
+  <div ng-show="currentTab == 'colors'">
+    <table class="table table-condensed col-table">
+      <thead>
+      <th>zIndex</th>
+      <th>Label</th>
```

Comment:
technically the direct child of a thead has to be `<tr>` though the browser tends to fill in the missing pieces